setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="majortom_gateway",
+    version="0.0.2",
+    author="Jesse Coffey",
```

Comment:
Consider making the author `Kubos`, but I don't know if it matters.

Reply:
👍